Lib/lib2to3/tests/__init__.py

```diff
@@ -1,8 +1,11 @@
 # Author: Collin Winter
 
 import os
+import warnings
 
 from test.support import load_package_tests
 
 def load_tests(*args):
-    return load_package_tests(os.path.dirname(__file__), *args)
+    with warnings.catch_warnings():
+        warnings.simplefilter('ignore', DeprecationWarning)
```

Comment:
Could you instead use `warnings.filterwarnings` to filter only warnings from lib2to3? That way we'll still know if lib2to3 uses a deprecated feature from some other module?

I haven't tried and I don't use warnings much, but I think this should do it:
```suggestion
        warnings.filterwarnings('ignore', category=DeprecationWarning, module='lib2to3')
```

Reply:
Let's roll the dice! 🍀